nodewatcher/core/generator/cgm/devices.py

```diff
@@ -529,6 +529,11 @@ class DeviceBase(object):
     radios = None
     switches = None
     usb = False
+    usb = None
+    usb = USB_2
+    usb = USB_3
+    USB_2 = 'usb-2'
```

Comment:
Don't define constants in `DeviceBase` class. I would suggest you create a new class, e.g. `USBProtocol` that would hold these two constants. Don't forget a documentation block.